homeassistant/components/modbus_sunspec/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "modbus_sunspec",
```

Comment:
We should probably instead add this as a config option to the modbus integration.

What do you say @balloob?

Reply:
Hi Martin,

Thanks for the review. Since raising the PR I've developed this a bit further but haven't tidied it up and pushed yet, I'll try to do that today.

One thing I've done is put a bit more context into the component so it knows the standard register locations of various sensors in a sunspec compliant device. Making it a bit higher level than it was might be an argument against rolling this into the modbus component.

Instead of end users needing to read the spec and find the offsets for various sensors they can just list the sensors in their configuration, this was inspired by reading other components in home assistant.

Here is a sample configuration for that style of component.
```
sensor:
  - platform: modbus_sunspec
    scan_interval: 1
    model:
    - name: meter
      base_register: 40188
      slave: 1
      sensors:
      - ac_current
        ac_power
        dc_current
        dc_power
        dc_voltage
        heat_sink_temp
    - name: inverter
      base_register: 40069
      slave: 1
      sensors:
      - ac_current
        ac_power
        ac_voltage
        ac_frequency
```

Reply:
I agree with Martin that we should try adding this to the modbus integration, since it's a spec to help understand the registers. You could add a `type` to the [sensor platform configuration](https://www.home-assistant.io/components/sensor.modbus/#configuration) and inside `setup_platform` you check the type and add the appropriate entities.